07.ls_object/file_etc.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'etc'
+
+class FileEtc
+  MODE_TABLE = {
+    '0' => '---',
+    '1' => '--x',
+    '2' => '-w-',
+    '3' => '-wx',
+    '4' => 'r--',
+    '5' => 'r-x',
+    '6' => 'rw-',
+    '7' => 'rwx'
+  }.freeze
+
+  attr_reader :name
+
+  def initialize(filename)
+    @name = filename
```

Comment:
```suggestion
  def initialize(name)
    @name = name
```

でもいいかも

https://bootcamp.fjord.jp/pages/readable-variable-name

Reply:
修正しました